cmd/globalAlignment/globalAlignment.go

```diff
@@ -40,6 +70,19 @@ func main() {
 	visualize := align.View(faOne.Seq, faTwo.Seq, aln)
 	fmt.Println(visualize)
 
+	//raven added this block to put visualized alignment output into MSA Fasta
+	visualizeOutput := ">" + faOne.Name + "\n" + strings.Split(visualize,"\n")[0] + "\n" + ">" + faTwo.Name + "\n" + strings.Split(visualize,"\n")[1]
```

Comment:
Great!  I would add another "\n" so that the last line of the file ends with an end-of-line character.

Reply:
Added!